### PR TITLE
chore: update package name

### PR DESCRIPTION
**Pull request by commoddity**
Updates package name from `pokt-foundation/portal-middleware` to `buildwithgrove/path`.

### CODE REVIEW

File: config/morse/signed_aat.go [hunk 1]
```diff
@@ -3,8 +3,8 @@ package morse
 import (
 	"fmt"
 
+	"github.com/buildwithgrove/path/config/utils"
```
Comment (adshmh) [inline config/morse/signed_aat.go:6]: Could you please update all imports so self-imports are at the bottom?

Comment (commoddity) [reply config/morse/signed_aat.go:6]: Sure.

**Review by adshmh — CHANGES_REQUESTED** [review]
Thank you for the PR, just one blocker.

File: config/morse/signed_aat.go (hunk 1 above)
Comment (commoddity) [reply config/morse/signed_aat.go]: > Could you please update all imports so self-imports are at the bottom?

Done.

**Review by adshmh — APPROVED** [review]

**Pull request merged**